src/web/mjs/connectors/CizgiRomanArsivi.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class CizgiRomanArsivi extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'cizgiromanarsivi';
+        super.label = 'Çizgi Roman Arşivi (CizgiRomanArsivi)';
+        this.tags = [ 'manga', 'english' ];
```

Comment:
I checked random content and it was `turkish` not `english`
May use `multi-lingual` if chapters are in different languages